triton-vm/src/table/hash_table.rs

```diff
@@ -344,6 +88,7 @@ impl Evaluable for ExtHashTable {
                 (round_constant_idx + round_constant_offset).into();
             evaluated_consistency_constraints.push(
                 round_number
+                    * (round_number - XFieldElement::from(9))
```

Comment:
```suggestion
                    * (round_number
                        - XFieldElement::new_const(BFieldElement::new(NUM_ROUNDS as u64 + 1)))
```
Hopefully, there's a better way to go from `usize` to `XFieldElement`.